builder/cpp/deps.bzl

```diff
@@ -7,3 +7,59 @@ def deps():
         strip_prefix = "googletest-release-1.12.1",
         urls = ["https://github.com/google/googletest/archive/refs/tags/release-1.12.1.zip"],
     )
+
+    http_archive(
+        name = "gherkin_cpp",
+        urls = ["https://github.com/krishnangovindraj/gherkin/archive/refs/heads/main.zip"],
+        strip_prefix = "gherkin-main/cpp",
+        sha256 = "29e098c87c97cdd43b1449af2d9cb05e9b76359e4d8028647671fff7fffb85e2",
+        build_file_content = """
+cc_library(
+  name = "gherkin-lib",
+  deps = ["@nlohmann_json//:json", "@cucumber_messages//:cucumber-messages-lib"],
+  hdrs = glob(["include/**/*.hpp"]),
+  strip_include_prefix = "include",
+  srcs = glob(["src/lib/gherkin/**/*.cpp"]),
+  copts = ["--std=c++17"],
+  visibility= ["//visibility:public"],
+)
+        """
+    )
+
+    http_archive(
+        name = "cucumber_messages",
+        urls = ["https://github.com/krishnangovindraj/cucumber-messages/archive/refs/heads/main.zip"],
```

Comment:
These point to my fork because there's a fix in it to make it clang compatible.
Do we want to
1. Fork it to vaticle & refer by commit?
2. Contribute back to the parent repository (Long term, I guess.)